pages/management/PKPToolsHandler.inc.php

```diff
@@ -160,4 +166,101 @@ public function resetPermissions($args, $request)
         echo json_encode(true);
         die;
     }
+
+    public function jobs($args, $request)
+    {
+        $this->setupTemplate($request, true);
+
+        $templateMgr = TemplateManager::getManager($request);
+
+        $queuedJobs = $this->buildQueuedJobsTable();
+
+        $totalQueuedJobs = $queuedJobs['total'];
+        $queuedData = $queuedJobs['data'];
+
+        $state = [
+            'components' => [
+                $queuedData->id => $queuedData->getConfig(),
+            ]
+        ];
+
+        $templateMgr->setState($state);
+
+        $templateMgr->assign([
+            'pageComponent' => 'JobsPage',
+            'pageTitle' => __('navigation.tools.jobs'),
+            'totalQueuedJobs' => $totalQueuedJobs
+        ]);
+
+        $templateMgr->display('management/tools/jobs.tpl');
+    }
+
+    /**
+     * Build the Queued Jobs Table
+     */
+    protected function buildQueuedJobsTable(): array
+    {
+        $dateFormatShort = Application::get()
+            ->getRequest()
+            ->getContext()
+            ->getLocalizedDateTimeFormatShort();
+
+        $jobsQuery = DB::table('jobs')
+            ->whereNotNull('queue')
+            ->whereNull('reserved_at');
+
+        $totalQueuedJobs = $jobsQuery->count();
+
+        $queuedJobsItems = $jobsQuery->get()
+            ->toArray();
```

Comment:
Since you're just doing a `foreach`, the `toArray` shouldn't be necessary.

Reply:
I've added it for debugging purposes, and forgot to remove. :facepalm: